package-lock.json

```diff
@@ -4,16 +4,6 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "2-thenable": {
```

Comment:
Just to double check, are these dependencies all unused?

Reply:
not entirely sure how these came to be - if they're in our `deps` and we can't find a relevant import in our codebase, then i'm assuming they're unused (subpackages should specify their own deps)

Reply:
removed all extra dpes